ansible/files/multinode.sh

```diff
@@ -0,0 +1,388 @@
+  fi
+
+  if [[ "$(sudo docker image ls)" == *"kayobe"* ]]; then
+    echo "Image already exists skipping docker build"
+  else
+    sudo DOCKER_BUILDKIT=1 docker build \
+      --network host \
+      --build-arg BASE_IMAGE=$BASE_IMAGE \
+      --file ${config_directories[kayobe]}/.automation/docker/kayobe/Dockerfile \
+      --tag kayobe:latest \
+      ${config_directories[kayobe]}
+  fi
+}
+
+function run_tempest() {
+  # Run Tempest test suite. Return non-zero if any tests failed.
+
+  tempest_dir="$HOME/tempest-artifacts"
+
+  seed_ssh=$(get_seed_ssh)
+
+  git -C ${config_directories[kayobe]} submodule init
+  git -C ${config_directories[kayobe]} submodule update
+
+  build_kayobe_image
+
+  set +x
+  export KAYOBE_AUTOMATION_SSH_PRIVATE_KEY=$(cat ~/.ssh/id_rsa)
+  set -x
+
+  if [[ -d $tempest_dir ]]; then
+    tempest_backup=${tempest_dir}.$(date --iso-8601=minutes)
+    echo "Found previous Tempest test results"
+    echo "Moving to $tempest_backup"
+    mv $tempest_dir $tempest_backup
+  fi
+
+  # Remove any previous kayobe_tempest container
+  sudo docker rm kayobe_tempest || true
+
+  # Run tempest
+  sudo -E docker run \
+    --name kayobe_tempest \
+    --detach --network host \
+    -v ${config_directories[kayobe]}:/stack/kayobe-automation-env/src/kayobe-config \
+    -v $tempest_dir:/stack/tempest-artifacts \
+    -e KAYOBE_ENVIRONMENT -e KAYOBE_VAULT_PASSWORD -e KAYOBE_AUTOMATION_SSH_PRIVATE_KEY \
+    kayobe:latest \
+    /stack/kayobe-automation-env/src/kayobe-config/.automation/pipeline/tempest.sh \
+    -e ansible_user=stack
+
+  # During the initial deployment the seed node must receive the `gwee/rally` image before we can follow the logs.
+  # Therefore, we must wait a reasonable amount time before attempting to do so.
+  sleep 360
+
+  if ! ssh -oStrictHostKeyChecking=no ${seed_ssh} 'sudo docker logs --follow tempest'; then
+    echo "Failed to follow Tempest container logs after waiting 360 seconds"
+    echo "Ignoring - this may or may not indicate an error"
+  fi
+
+  # Wait for Kayobe Tempest pipeline to complete to ensure artifacts exist.
+  kayobe_tempest_rc="$(sudo docker container wait kayobe_tempest)"
+  if [[ $kayobe_tempest_rc != "0" ]]; then
+    echo "Failed running kayobe_tempest container. Output:"
+    sudo docker logs kayobe_tempest
+  fi
+  sudo docker rm kayobe_tempest
+
+  if [[ ! -f $tempest_dir/failed-tests ]]; then
+    echo "Unable to find Tempest test results in $tempest_dir/failed-tests"
+    return 1
+  fi
+
+  if [[ $(wc -l < $tempest_dir/failed-tests) -ne 0 ]]; then
+    echo "Some Tempest tests failed"
+    return 1
+  fi
+
+  echo "Tempest testing successful"
+}
+
+function run_tests() {
+  rc=0
+  if ! run_tempest; then
+    rc=1
+  fi
+  return $rc
+}
+
+function deploy_full() {
+  # End-to-end deployment and testing.
+
+  deploy_seed
+  deploy_overcloud
+  if kayobe configuration dump --host wazuh-manager --var-name group_names | grep wazuh-manager &>/dev/null; then
```

Comment:
It might be a bit clearer to use:
```suggestion
  if kayobe configuration dump -l wazuh-manager &>/dev/null; then
```
Which will error if no `wazuh-manager` exists

Reply:
Have you tried it?

Reply:
nope!

Reply:
I tried your command and it failed on a system with wazuh :)

Reply:
:(